docs/contributing.rst

```diff
@@ -60,6 +60,13 @@ Prepare Resolwe for development::
 
 .. _Resolwe's git repository: https://github.com/genialis/resolwe
 
+Signed commits are required in the Resolwe upstream repository.
```

Comment:
I would move this to a separate section named **Submitting changes to upstream** and put it just before the **Preparing release** section.